vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -1093,7 +1093,6 @@ testWasmRuntimesInstalled() {
   local test="testWasmRuntimesInstalled"
   local wasm_runtimes_path=${1}
   local shim_version=${2}
-  shim_version="v${shim_version}"
```

Comment:
nit (not blocking): this could be combined with the previous line - no need to assign to a variable and then immediately change it.

Reply:
this line is deleted.